src/Jira/Api/Client/PHPClient.php

```diff
@@ -201,11 +201,13 @@ protected function doSendRequest($url, array $context)
 	{
 		$this->_lastErrorMessage = '';
 
-		set_error_handler(array($this, 'errorHandler'));
+		/** @var callable $callable */
+		$callable = array($this, 'errorHandler');
+		set_error_handler($callable);
```

Comment:
Why you've changed this? The original code version looks working to me.

Reply:
for PHPStan, as this is a deprecated callable

Reply:
I see. Next time you can post links such as https://php.watch/versions/8.2/partially-supported-callable-deprecation to same me some time figuring this for myself.